### ISSUE TITLE
Success Message Is Not Announced to Screen Reader Users

### ISSUE THREAD

**Issue by MajoMaldonado** [equalizedigital/accessibility-checker#1759]
Location / URL: https://edac-vpat.instawp.co/wp-admin/admin.php?page=accessibility_checker_settings&tab=accessibility-reports 
Screenshot: 

<img width="777" height="512" alt="Image" src="https://github.com/user-attachments/assets/764fde78-3ed4-4838-95a8-407ee1afadad" />

WCAG S.C: 4.1.3 Status Messages (Level AA)
Issue Description: After entering a valid license key, the page refreshes and a success message is displayed indicating that the site has been registered successfully. However, the message is not announced by screen readers after the page reloads. Success status messages should be programmatically identified so assistive technologies automatically announce them when they appear, without requiring users to manually navigate to the message.

**Comment by amberhinds** [comment]
I think this is caused by WordPress core, and the only solution to it would be to build this using JS rather than the WP core settings functions.
